helmfile/charts/calico-accountant/templates/rbac.yaml

```
@@ -0,0 +1,50 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: calico-accountant
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: calico-accountant-privileged
```

```suggestion
  name: calico-accountant
```
Nit: feels like "*-privileged" is only fitting if the role only handles the PSP access.